docs/modules/ROOT/pages/getting-started/filters-and-projections.adoc

```diff
@@ -0,0 +1,191 @@
+
+The resulting query will look like this:
+
+
+```cypher
+MATCH(m:Movie)
+WHERE (m.title = "The Matrix" AND m.released < 2000)
+RETURN m.title, m.tagline, m.released
+```
+
+NOTE: This query will return all movies with the name `"The Matrix"` released before `2000`. This query is based on the default example database in Neo4j, and will return the classic 1999 film **The Matrix**. 
+
+
+== Filtering
+
+When creating a `MATCH` clause, it is possible to add subclauses to it in a similar way to how you added the `RETURN` statement. Change the `Cypher.Match` instance to add a `WHERE` subclause with the method `.where`:
+
+```javascript
+const clause = new Cypher.Match(movieNode)
+    .where(Cypher.eq(movieNode.property("title"), new Cypher.Param("The Matrix")))
+    .return(movieNode);
+```
+
+There is quite a bit to unwrap here. First, you'll notice that the `.where` method takes the result of the function `Cypher.eq`. This function is the **eq**uality (`=`) operator that compares the property `title` and the string `The Matrix`.
+
+`Cypher.eq` takes two arguments to compare. Our first argument will be the property `title`. Because our title is a property of our movieNode variable, the `.property` method of that node will ensure that the property correctly references the movieNode variable.
+
+The second parameter is `"The Matrix"`, but instead of using a plain string, the object `Cypher.Param` class will take care of turning that into a parameter so we can safely put user input here.
+
+
+.Why `Cypher.eq` doesn't need `new`?
+[NOTE]
+====
+Cypher Builder treats Variables (like nodes and relationships) and Clauses as classes, as these are intended to be assigned to a JavaScript variable to be reused, composed and built.
+Cypher Functions and Procedures, however, are intended to be used a single time, as part of a larger query, so Cypher Builder treats these as simple functions, making it closer to how these work in Cypher.
+====
+
+=== Cypher and params
+
+You can run your script again (`node main.js`), and should get the following Cypher:
+
+```cypher
+MATCH (this0:Movie)
+WHERE this0.title = $param0
+RETURN this0
+```
+
+Success! The movie is now filtered by its title to a param. The param, however, is missing from the output. You'll need to tweak the build process at the end of the script to get the parameters as well:
+
+```javascript
+const { cypher, params } = match.build();
+console.log(cypher);
+console.log(params);
+```
+
+Now, executing again, we get all we need to run this query in Neo4j:
+
+```cypher
+MATCH (this0:Movie)
+WHERE this0.title = $param0
+RETURN this0
+```
+
+```javascript
+{ param0: 'The Matrix' }
+```
+
+[TIP]
+====
+Note that `new Param` already takes care of assigning a param name and correctly matching the value to that parameter in the Cypher query. If a `Param` instance is used multiple times within a query, it will always reference the same parameter (`$param0` in this case).
```

Comment:
The absence of explanation about `new Param` before is making this note a bit difficult to understand